streetlight/streetlight_destinations.sql

```diff
@@ -0,0 +1,28 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
+-- proj: 3857
+----------------------------------------
+DROP TABLE IF EXISTS neighborhood_streetlight_destinations;
+CREATE TABLE generated.neighborhood_streetlight_destinations (
+    id SERIAL PRIMARY KEY,
+    geom geometry(multipolygon,4326),
+    name TEXT,
+    blockid10 TEXT,
+    is_pass INT
+);
+
+INSERT INTO neighborhood_streetlight_destinations (
+    blockid10,
+    name,
+    geom,
+    is_pass
+)
+SELECT  blocks.blockid10,
+        blocks.blockid10,
+        ST_Transform(blocks.geom,4326),
+        0
+FROM    neighborhood_census_blocks blocks,
+        neighborhood_zip_codes zips
+WHERE   zips.zip_code = '02138'
```

Comment:
This script isn't in use by the core tool at the moment but it should be updated. It will eventually be used with the StreetLightData option that is still being evaluated by PFB but likely to be OK'd.